backend/config.js

```diff
@@ -186,20 +187,31 @@ var environments = {
                     email: 'user1@mail.net',
                     roleID: 3,
                     password: 'testuser1',
+                    authId: 1,
                 },
                 {
                     firstName: 'User2',
                     lastName: 'Test',
                     email: 'user2@mail.net',
                     roleID: 3,
                     password: 'testuser2',
+                    authId: 2,
                 },
                 {
                     firstName: 'User3',
                     lastName: 'Test',
                     email: 'user3@mail.net',
                     roleID: 3,
                     password: 'testuser3',
+                    authId: 3,
+                },
+                //Added this user to be created on the auth service
+                {
+                    firstName: 'SYSMessageUser',
```

Comment:
We need this on the auth service in order for greyscale to run. So I added it here so we don't have to always do it manually. With this, running `user.integration.js` creates it on the auth service for us.